components/application-mgt/org.wso2.carbon.identity.application.mgt/src/main/java/org/wso2/carbon/identity/application/mgt/dao/impl/ApplicationDAOImpl.java

```diff
@@ -1309,6 +1310,17 @@ private OutboundProvisioningConfig getOutboundProvisioningConfiguration(int appl
         return outBoundProvisioningConfig;
     }
 
+    private boolean updateCheck(LocalAndOutboundAuthenticationConfig localAndOutboundAuthConfig) {
```

Comment:
check formatting issue

Reply:
can we rename this method check to a meaningful one?

Reply:
also did we tested migration and rest APIs, also what happen when a user authenticates with migrated app ? what is the error thrown?

Reply:
Tested with the REST API for that scenario this was thrown as expected. For the migration scenario, migration happens successfully but when trying to use the service provider which has script based adaptive auth a NullponiterException is thrown, currently fixing that issue, to show a meaningful message.